colosseum/parser.py

```diff
@@ -126,3 +128,133 @@ def color(value):
                 pass
 
     raise ValueError('Unknown color %s' % value)
+
+
+##############################################################################
+# Font handling
+##############################################################################
+def _parse_font_property_part(value, font_dict):
+    """Parse font shorthand property part for known properties."""
+    from .constants import (FONT_SIZE_CHOICES, FONT_STYLE_CHOICES, FONT_VARIANT_CHOICES,
+                            FONT_WEIGHT_CHOICES, LINE_HEIGHT_CHOICES, NORMAL)
+    font_dict = font_dict.copy()
```

Comment:
I don't follow you here. Could you elaborate?

Reply:
So - this function accepts a full font dictionary as an argument, takes a copy, and then returns the copy. I'm saying that the function doesn't need to accept the dictionary, just the value; it can parse that value, and return the dictionary of *new* values, the calling function can then merge into the "master" definition as required (and as allowed - since it's possible you could have conflicting definitions, like the "bigger smaller Helvetica" example that I gave)

Reply:
Gotcha